org.moonsoft.MoonPlayer.json

```diff
@@ -0,0 +1,195 @@
+{
+  "app-id": "org.moonsoft.MoonPlayer",
+  "runtime": "org.kde.Platform",
+  "runtime-version": "5.11",
+  "sdk": "org.kde.Sdk",
+  "command": "moonplayer",
+  "rename-icon": "moonplayer",
+  "rename-desktop-file": "moonplayer.desktop",
+  "finish-args": [
+      "--share=ipc",
+      "--socket=x11",
+      "--socket=wayland",
+      "--device=all",
+      "--share=network",
+      "--socket=pulseaudio",
+      "--filesystem=host",
+      "--env=LC_NUMERIC=C"
+  ],
+  "modules": [
+    {
+      "name": "moonplayer",
+      "buildsystem": "qmake",
+      "sources": [
+        {
+          "type": "archive",
+          "url": "https://github.com/coslyk/moonplayer/archive/2.1.1.tar.gz",
+          "sha256": "67c74681d12c10a4ecc4c2bad22006e22bb804b546ba470cac0408b1f02f1f58"
+        },
+        {
+          "type": "shell",
+          "commands": [
```

Comment:
Why are these not fixed upstream?

Reply:
It seems that there is some problems with the package libqtermwidget5-0-dev in Ubuntu 18.04, the pkg-config doesn't work for it...

Reply:
Is QMake not capable of handling it? I'm not sure why that should be a blocker. There were also other workarounds.

Reply:
I found a new workaround to avoid handling it in json file. I have updated it to upstream. Thanks for suggestion